### DISCUSSION TITLE
I got this error when I deployed advanced blog

### DISCUSSION THREAD

**Discussion by bugsecuser** [avitorio/outstatic#214] [Q&A] [answered]
![Error-advanced-blog](https://github.com/avitorio/outstatic/assets/133862122/b61ffa5c-4a96-48c9-b8bc-43bff190541d)

How to fix this? @avitorio 

Thanks,

**Comment by avitorio** [comment]
@bugsecuser Please, give me more info on how to replicate this error. What steps did you take?

**Comment by bugsecuser** [comment]
@avitorio Thanks for the quick response!
Got this while building a new blog setup. Is there any "API Keys needed for installation", So how can it be done.

**Reply by avitorio** [reply]
I just confirmed the error, working on a fix. The only keys you need are your github keys as explained in the [documentation](https://outstatic.com/docs/getting-started).

**Comment by bugsecuser** [comment]
hey @avitorio i see the new fix, but still i get that error.

**Reply by avitorio** [reply, marked as answer]
Hey @bugsecuser , seems to be fixed now. Can you test it? Thanks!

**Comment by bugsecuser** [comment]
Hi @avitorio Yes, it's completely fine now! 👍

**Comment by growindiedev** [comment]
I'm still facing this issue. How to fix it?

**Reply by avitorio** [reply]
Check your version of Outstatic @growindiedev make sure you are using the latest

**Reply by avitorio** [reply]
Hey @growindiedev,
There was actually a type error on the advanced-blog example that has been fixed. You should be able to clone the project and deploy to vercel. Cheers!

**Reply by growindiedev** [reply]
Thanks @avitorio. Appreciate it : )
